relayer/chainproxy/portalLogs.go

```diff
@@ -19,28 +20,33 @@ const webSocketCloseMessage = "websocket: close 1005 (no status)"
 
 type PortalLogs struct {
 	newRelicApplication *newrelic.Application
+	MetricService       *metrics.MetricService
+	StoreMetricData     bool
 }
 
-func NewPortalLogs() (*PortalLogs, error) {
+func NewPortalLogs(storeMetricData bool) (*PortalLogs, error) {
```

Comment:
it should be decided based on environment variable.

Reply:
that is for differentiating btw server and portal_server, I added support for enabling or disabling this

Reply:
we may want to support it also for providers ("server") in the future, and until then we can just use the new env var to disable it. please remove the variable from the constructor, it's redundant with the env var